common/js_handle.go

```diff
@@ -191,10 +173,19 @@ func (h *BaseJSHandle) JSONValue() string { // TODO: return error
 		if result, _, err = action.Do(cdp.WithExecutor(h.ctx, h.session)); err != nil {
 			k6ext.Panic(h.ctx, "getting properties for JS handle: %w", err)
 		}
-		remoteObject = result
+		res, err := parseConsoleRemoteObject(h.logger, result)
+		if err != nil {
+			return "", fmt.Errorf("extracting value from result: %w", err)
+		}
+
+		return res, nil
```

Comment:
This part was refactored for clarity. Can we save `result` to `remoteObject` and call `parseConsoleRemoteObject` once?

Reply:
I would think that makes sense. Not sure why that was not being done before, but I don't think that has any issues.
Also I noticed we are returning two different error messages (`value from result` | `value from remote object`), shouldn't these be the same as we are executing the same action?

Reply:
Yes, I agree. We don't have to differentiate between these errors. If there was an error retrieving the data via `ObjectID`, there will already be a clarifying error.

Reply:
I guess the two error messages made sense for debug purposes to help clarify which conditional branch the method was working on. The [refactor](https://github.com/grafana/xk6-browser/blob/f83f34fe646f8cb0f94733a1657c8e7e3da04b19/common/js_handle.go#L183-L197) makes more sense still. It has been added to b77858857d1accc0509bddbb6e6efe6d816f7ad3